Dockerfile

```diff
@@ -1,80 +1,59 @@
-FROM alpine:3.11 AS builder
-
-RUN apk add --no-cache \
-      bash \
-      bubblewrap \
-      build-base \
-      ca-certificates \
-      coreutils \
-      curl \
-      git \
-      m4 \
-      ncurses-dev \
-      opam \
-      ocaml \
-      ocaml-compiler-libs \
-      patch \
-      rsync \
-      tar \
-      xz \
-    ;
-
-# Disable sandboxing
-RUN set -ex; \
-    echo 'wrap-build-commands: []' > ~/.opamrc; \
-    echo 'wrap-install-commands: []' >> ~/.opamrc; \
-    echo 'wrap-remove-commands: []' >> ~/.opamrc; \
-    echo 'required-tools: []' >> ~/.opamrc;
+FROM alpine:3.16
```

Comment:
We can switch to `ubuntu` if we run into any issues. Alpine is just more convenient to remove build deps. I'll accept a PR to migrate if the final image size is comparable.

I doubt a possibility of slight version shifts in system dependencies when rebuilding the image will be a problem for Coq kata, but we'll see.